docs/reconfigurator-dev-guide.adoc

```diff
@@ -1343,4 +1361,13 @@ Sled SimGimlet00
 ...
 ```
 
+If you're running `omicron-dev run-all`, you'll need to use the `--dns-server` flag to specify
+the DNS server `omdb` will use. You can find this information within the `omicron-dev run-all` output:
+
+```
+...
+omicron-dev: internal DNS:          [::1]:63673
+...
+```
```

Comment:
I realise this is mentioned elsewhere, but if you're like me and not necessarily read the doc in order, this information is helpful here too 😅 

Happy to remove if it feels too redundant though

Reply:
I think it's actually in two other places in this file already.  It might be worth cleaning up but it's not a big deal and definitely not a blocker for this.